lib/local_service_cleaner.rb

```diff
@@ -0,0 +1,48 @@
+require 'set'
+
+class LocalServiceCleaner
+  def initialize(input_data = LocalServiceImporter.fetch_data)
+    @input_data = input_data
+  end
+
+  DetectionResult = Struct.new(:local_service, :"in_csv?", :"in_local_transaction?")
```

Comment:
Prove that I'm a messy developer who doesn't tidy up vestigial tails of alternative versions I decided against showing to the world.  Will delete.